src/nidigital/metadata/functions.py

```diff
@@ -961,8 +973,10 @@
                 'type': 'ViSession'
             },
             {
+
                 'direction': 'in',
                 'name': 'site',
+                'python_api_converter_name': 'convert_site_string',
```

Comment:
`FetchHistoryRAMCycleInformation`, `FetchHistoryRAMCyclePinData`, and `FetchHistoryRAMScanCycleNumber` are private. We could use the converter only in the public method - `FancyFetchHistoryRAMCycleInformation` and assume private methods accept `site` as "siteN".

Reply:
Changed private functions to not use converter